packages/@aws-cdk/aws-scheduler-targets-alpha/lib/codebuild-start-build.ts

```diff
@@ -0,0 +1,37 @@
+import { ISchedule, IScheduleTarget } from '@aws-cdk/aws-scheduler-alpha';
+import { Names } from 'aws-cdk-lib';
+import { IProject } from 'aws-cdk-lib/aws-codebuild';
+import { IRole, PolicyStatement } from 'aws-cdk-lib/aws-iam';
+import { ScheduleTargetBase, ScheduleTargetBaseProps } from './target';
+import { sameEnvDimension } from './util';
+
+/**
+ * Use an AWS CodeBuild as a target for AWS EventBridge Scheduler.
+ */
+export class CodeBuildStartBuild extends ScheduleTargetBase implements IScheduleTarget {
+  constructor(
+    private readonly project: IProject,
+    private readonly props: ScheduleTargetBaseProps,
```

Comment:
```suggestion
    private readonly props: ScheduleTargetBaseProps = {},
```

Reply:
an aside: why did we call this props if they were all optional lol

Reply:
aaah yes. a default value totally makes sense. thanks for pointing out haha.